source/isaaclab/isaaclab/managers/event_manager.py

```diff
@@ -384,7 +384,7 @@ def _prepare_terms(self):
             self._mode_term_cfgs[term_cfg.mode].append(term_cfg)
 
             # check if the term is a class
-            if isinstance(term_cfg.func, ManagerTermBase):
+            if isinstance(term_cfg.func, type(ManagerTermBase)):
```

Comment:
ManagerTermBase is already a class so it is the right type check. Doing a type on top of it will return a python metaclass object AFAIK which won't make sense.

Can you please provide a repro / unit test if you're seeing issues.

Reply:
Will do.

Reply:
Done.